openslides/agenda/static/js/agenda/site.js

```diff
@@ -613,6 +687,30 @@ angular.module('OpenSlidesApp.agenda.site', [
     }
 ])
 
+.factory('AgendaCsvExport', [
```

Comment:
Please move this factory to a speparate angular module in a separate file (like for PDF).